homeassistant/components/unifi/update.py

```diff
@@ -79,6 +80,9 @@ def __init__(self, device, controller):
 
         self.device = self._item
 
+        if self.controller.site_role == "admin":
```

Comment:
I double checked how this is down in another integration and if there are variations it should all be set in the init https://github.com/home-assistant/core/blob/027cdbdb3893f3341629b7394e8bd0e24c6cd325/homeassistant/components/deconz/climate.py#L121

So one final change is needed. Sorry

Reply:
Sure, no problem